clio-server/src/main/scala/org/broadinstitute/clio/server/service/SchemaService.scala

```diff
@@ -116,6 +116,7 @@ object SchemaService {
         case t if t =:= typeOf[Option[Long]]              => (false, aLong)
         case t if t =:= typeOf[Option[String]]            => (false, aString)
         case t if t =:= typeOf[Option[OffsetDateTime]]    => (false, aTime)
+        case t if t =:= typeOf[Option[UBamFileStatus]]    => (false, aString)
```

Comment:
Instead of adding this new line, can we just change:
```scala
case t if t =:= typeOf[TransferReadGroupLocation] => (true, aString)
```
in the lines above to:
```scala
case t if t =:= typeOf[EnumEntry] => (true, aString)
```
to cover all the enums we might add?